x/auth/types/stdtx_test.go

```diff
@@ -68,25 +65,26 @@ func TestStdTx(t *testing.T) {
 
 func TestStdSignBytes(t *testing.T) {
 	type args struct {
-		chainID  string
-		accnum   uint64
-		sequence uint64
-		fee      StdFee
-		msgs     []sdk.Msg
-		memo     string
+		chainID       string
+		accnum        uint64
+		sequence      uint64
+		timeoutHeight uint64
+		fee           StdFee
+		msgs          []sdk.Msg
+		memo          string
 	}
 	defaultFee := NewTestStdFee()
 	tests := []struct {
 		args args
 		want string
 	}{
 		{
-			args{"1234", 3, 6, defaultFee, []sdk.Msg{testdata.NewTestMsg(addr)}, "memo"},
-			fmt.Sprintf("{\"account_number\":\"3\",\"chain_id\":\"1234\",\"fee\":{\"amount\":[{\"amount\":\"150\",\"denom\":\"atom\"}],\"gas\":\"100000\"},\"memo\":\"memo\",\"msgs\":[[\"%s\"]],\"sequence\":\"6\"}", addr),
+			args{"1234", 3, 6, 10, defaultFee, []sdk.Msg{testdata.NewTestMsg(addr)}, "memo"},
+			fmt.Sprintf("{\"account_number\":\"3\",\"chain_id\":\"1234\",\"fee\":{\"amount\":[{\"amount\":\"150\",\"denom\":\"atom\"}],\"gas\":\"100000\"},\"memo\":\"memo\",\"msgs\":[[\"%s\"]],\"sequence\":\"6\",\"timeout_height\":\"10\"}", addr),
```

Comment:
Can we add another test case that verifies that timeout_height is not part of the `SignDoc` when set to 0?